src/WingetCreateCLI/Commands/UpdateLocaleCommand.cs

```diff
@@ -0,0 +1,291 @@
+                            (commandEvent.IsSuccessful = await this.GitHubSubmitManifests(
+                                originalManifests,
+                                this.GetPRTitle(originalManifests, null, nameof(UpdateLocaleCommand))))
+                            : false;
+                    }
+                    else
+                    {
+                        Logger.WarnLocalized(nameof(Resources.SkippingPullRequest_Message));
+                    }
+                }
+                else
+                {
+                    return false;
+                }
+
+                return await Task.FromResult(commandEvent.IsSuccessful = true);
+            }
+            catch (Octokit.NotFoundException e)
+            {
+                Logger.ErrorLocalized(nameof(Resources.Error_Prefix), e.Message);
+                Logger.ErrorLocalized(nameof(Resources.OctokitNotFound_Error));
+                return false;
+            }
+            finally
+            {
+                TelemetryManager.Log.WriteEvent(commandEvent);
+            }
+        }
+
+        private static object PromptAllLocalesAsMenuSelection(Manifests originalManifests)
+        {
+            Dictionary<string, object> localeMap = new Dictionary<string, object>
+            {
+                { originalManifests.DefaultLocaleManifest.PackageLocale + " (" + Resources.DefaultLocale_MenuItem + ")", originalManifests.DefaultLocaleManifest },
+            };
+
+            foreach (var locale in originalManifests.LocaleManifests)
+            {
+                localeMap.Add(locale.PackageLocale, locale);
+            }
+
+            string selectedLocale = Prompt.Select(Resources.SelectExistingLocale_Message, localeMap.Keys.ToList());
+            return localeMap[selectedLocale];
+        }
+
+        private static void DisplayUpdatedLocales(Manifests manifest)
+        {
+            Logger.DebugLocalized(nameof(Resources.GenerateUpdatedLocalePreview_Message));
+            if (manifest.DefaultLocaleManifest != null)
+            {
+                DisplayDefaultLocaleManifest(manifest.DefaultLocaleManifest);
+            }
+
+            if (manifest.LocaleManifests != null)
+            {
+                DisplayLocaleManifests(manifest.LocaleManifests);
+            }
+        }
+
+        private Manifests PromptAndUpdateExistingLocales(Manifests originalManifests)
+        {
+            Manifests updatedLocales = new Manifests()
+            {
+                SingletonManifest = null,
+                VersionManifest = null,
+                InstallerManifest = null,
+            };
+
+            List<string> defaultPromptPropertiesForUpdateLocale = new()
+            {
+                nameof(LocaleManifest.PackageName),
+                nameof(LocaleManifest.Publisher),
+                nameof(LocaleManifest.License),
+                nameof(LocaleManifest.ShortDescription),
+            };
```

Comment:
How did you come up with these defaults? Since we are dealing with locale, I would assume almost every property would need to be updated. An alternative could be to prompt for every field that contains an existing value except the ones that can't be changed (PackageId, PackageVersion, etc.).

Reply:
I made these defaults for new-locale for the user to quickly create a very basic/barebone locale manifest, and then prompt if user wants to add additional fields. I made those properties default that are marked as "required" in case of a default locale manifest.

For the update-locale process, it might be more intuitive to prompt for all properties, as it wouldn't be evident beforehand which properties the user is likely to update. Users can also quickly accept values for other properties that don't interest them.